idfive-component-library/src/components/program-finder/program-finder.hbs

```diff
@@ -7,7 +7,12 @@
       <h3 class="program-finder__nav-heading">Undergraduate and Graduate Programs</h3>
   {{/if}}
   <div class="program-finder-app">
-    <program-finder env="prod" degree="441" school="66" subject="" />
+    <program-finder
+      env="prod"
+      degree="441"
+      school="66"
+      subject=""
+    />
```

Comment:
Are degree and school defaults needed/wanted, shouldnt it default to all?

Reply:
This is just for demo purposes in fractal, those props automatically select a degree, school, or subject upon load if a value is passed to them. If they're blank it just defaults to all.